### PR TITLE
Feat: Add changing default values to risk calculation. Add tests

### PR DESCRIPTION
**Pull request by StanislawMazowieckiAriane**

### CODE REVIEW

File: src/risk/index.ts [hunk 1]
```diff
@@ -44,13 +38,21 @@ const defaultWeights: Weights = {
...
 };
 
+const updateWeights = (newWeights: Weights) => {
+  Object.assign(defaultWeights, newWeights);
+};
+
+const updateRiskLevels = (newRiskLevels: RiskLevels) => {
+  Object.assign(defaultRiskLevels, newRiskLevels);
+};
+
```
Comment (se7enarianelabs) [inline src/risk/index.ts:55]: In my opinion, the risk functions should be stateless. This means you should be able to provide your weights and riskLevels through optional parameters. If the parameters are omitted, the function should use default values.

File: src/risk/index.ts [hunk 2]
```diff
@@ -61,10 +63,7 @@ const calculateRiskScoreFromData = (metadata: Metadata): RiskResult => {
   };
 };
 
-const calculateRiskScoreFromTokenId = async (
-  tokenId: string,
-  network: string = 'mainnet'
-): Promise<RiskResult> => {
+const calculateRiskScoreFromTokenId = async (tokenId: string, network: string = 'mainnet'): Promise<RiskResult> => {
   const uri =
     network === 'mainnet'
```
Comment (se7enarianelabs) [inline src/risk/index.ts:68]: The SDK supports previewnet and localnodes as well

**Review by se7enarianelabs — CHANGES_REQUESTED** [review]

File: src/risk/index.ts (hunk 1 above)
Comment (StanislawMazowieckiAriane) [reply src/risk/index.ts:55]: But it is. It has default values. Only if the user want to customize them, he uses one of the update functions

File: src/risk/index.ts (hunk 2 above)
Comment (StanislawMazowieckiAriane) [reply src/risk/index.ts]: Done

File: src/risk/index.ts (hunk 1 above)
Comment (StanislawMazowieckiAriane) [reply src/risk/index.ts]: Done

File: src/risk/index.ts [hunk 3]
```diff
@@ -63,11 +65,28 @@ const calculateRiskScoreFromData = (metadata: Metadata): RiskResult => {
...
+      uri = `https://previewnet.mirrornode.hedera.com/api/v1/tokens/${tokenId}/`;
+      break;
+    case 'localNode':
+      if (!localNodeURL) {
+        throw new Error('localNodeURL is required for local node network');
+      }
+      uri = `${localNodeURL}/api/v1/tokens/${tokenId}/`;
+      break;
+    default:
+      throw new Error('Invalid network');
```
[the remark this answers is not in the document]
Comment (StanislawMazowieckiAriane) [reply src/risk/index.ts:100]: Done

**Review by se7enarianelabs — APPROVED** [review]

**Review by michalrozekariane — APPROVED** [review]

**Pull request merged**